src/api-service/dev-deploy.ps1

```diff
@@ -0,0 +1,19 @@
+param (
+    [Parameter()]
+    [string] $app_dir="$PSScriptRoot",
+    [Parameter(Mandatory=$true)]
+    [string] $target
+)
+
+Set-Location "$app_dir\..\pytypes"
+python setup.py sdist bdist_wheel 
+Copy-Item dist/*.whl "$app_dir\__app__"
+Set-Location $app_dir
+Set-Location __app__
+New-Guid | ForEach-Object{$_.Guid} > onefuzzlib/build.id
+(Get-Content -path "requirements.txt") -replace 'onefuzztypes==0.0.0', './onefuzztypes-0.0.0-py3-none-any.whl' | Out-File "requirements.txt"
+func azure functionapp publish $target --python
+(Get-Content -path "requirements.txt") -replace './onefuzztypes-0.0.0-py3-none-any.whl', 'onefuzztypes==0.0.0' | Out-File "requirements.txt"
```

Comment:
It seems safer and more reliable to backup the file and restore.

Reply:
Keeping the same logic as the dev-deploy.sh to make the maintenance easier. we can address the comment in both script in a sperate PR